curriculum/challenges/_meta/workshop-piano/meta.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "Build a Piano",
```

Comment:
It might work better as Design a Piano. Or maybe Design a keyboard layout?

Reply:
I would go with "Design a Piano" but if you think the other option sounds better I'll trust you :+1:  @jdwilkin4

Reply:
Yeah, Design a Piano works for me